packages/plugin-ext/src/hosted/node/plugin-host.ts

```diff
@@ -104,5 +120,35 @@ process.on('message', async (message: string) => {
     }
 });
 
-const pluginHostRPC = new PluginHostRPC(rpc);
-pluginHostRPC.initialize();
+const container = new Container();
+container.bind(PluginContainerModuleLoader).toConstantValue((module: ContainerModule) => {
+    container.load(module);
+    const internalModule = module as InternalPluginContainerModule;
+    const pluginApiCache = internalModule.initializeApi?.(container);
+    if (pluginApiCache) {
+        // TODO
```

Comment:
See my other comment on the TODO, what do we need to do? What is the impact? Should we track that in a separate issue?